recipes-debian/AppArmor/apparmor_debian.bb

```diff
@@ -0,0 +1,204 @@
+#base recipe: meta-security/recipes-mac/AppArmor/apparmor_2.13.2.bb
+#base branch: warrior
+#base commit: 4f7be0d252f68d8e8d442a7ed8c6e8a852872d28
+
+SUMMARY = "AppArmor another MAC control system"
+DESCRIPTION = "user-space parser utility for AppArmor \
+This provides the system initialization scripts needed to use the \
+AppArmor Mandatory Access Control system, including the AppArmor Parser \
+which is required to convert AppArmor text profiles into machine-readable \
+policies that are loaded into the kernel for use with the AppArmor Linux \
+Security Module."
+HOMEAPAGE = "http://apparmor.net/"
+SECTION = "admin"
+
+LICENSE = "GPLv2 & GPLv2+ & BSD-3-Clause & LGPLv2.1+"
+LIC_FILES_CHKSUM = "file://${S}/LICENSE;md5=fd57a4b0bc782d7b80fd431f10bbf9d0"
+
+DEPENDS = "bison-native apr gettext-native coreutils-native"
+
+inherit pkgconfig autotools-brokensep update-rc.d python3native perlnative ptest cpan manpages systemd debian-package 
+
+SRC_URI += " \
+        file://disable_perl_h_check.patch \
+        file://crosscompile_perl_bindings.patch \
+        file://apparmor.rc \
+        file://functions \
+        file://apparmor \
+        file://apparmor.service \
+        file://run-ptest \
+        "
+
+PARALLEL_MAKE = ""
+
+require recipes-debian/sources/apparmor.inc
+
+DEBIAN_PATCH_TYPE = "nopatch"
```

Comment:
debian added some patches (https://sources.debian.org/patches/apparmor/2.13.2-10/). Is there any reasons to not apply them?

Reply:
Fixed to apply debian patch